### ISSUE TITLE
remove missing / PD stems from census whenever possible

### ISSUE THREAD

**Issue by teixeirak** [SCBI-ForestGEO/SCBImortality#93]
A lot of time is spent searching for PD (previously dead) / missing stems. We can speed things up if we remove as many as possible:

Guidelines:
- if a PD or missing tree was found dead on the ground in any previous census, it should be counted as PD and skipped.
- if a stem is recorded as missing 2 years in row, it should be dropped
- it is lower priority to search for PD and missing stems than to complete the census

Actions:
1. @emacmonigle, @jenajordan , and other surveyors should stop spending time searching for PD or missing stems. Time permitting, we can go back and try to find any that are problematic after the system is modified. 
2. we need to go back and review exactly how PD was applied in the past
3. @ValentineHerr / @emacmonigle , we'll want to modify the tests as follows :
- [x]  stems missed in the census that were previously PD/ missing are not flagged as an error or warning if they were ever recorded dead on the ground
- [x] stems missed in the census that were previously missing for 2 years are not flagged as an error or warning
- [x] stems that were missed in the census that were PD/ missing in the previous census are flagged as warning, as opposed to error. Time permitting, we can switch these back to errors and go back to verify their status.

**Comment by teixeirak** [comment]
It may make sense for @jess-shue to modify the form to avoid PD/missing stems that were dead down or confirmed missing, but I believe this would be fairly complicated, so perhaps not worth the effort.

**Comment by jenajordan** [comment]
@teixeirak, I am not with the iPad right now. Just clarifying, I believe on FF there is no choice to state "PD". I will assume I should choose dead not found? 

Also - for stems previously DS or dead not found, I believe FF is still requiring we enter lianas. Can we remove that? It will save a little time.

**Comment by ValentineHerr** [comment]
> * [ ] stems missed in the census that were previously PD/ missing are not flagged as an error or warning if they were ever recorded dead on the ground 

** Can you remind me where I should look to figure out if a tree was recorded as "dead on the ground"? -- is it okay if I consider ever as since 2 census ago?**

**Comment by teixeirak** [comment]
Going back 2 censuses ago would be a start, but best if we could get the full set of censuses. So anything that's ever been status DC. That would be in all_mort, correct? (I can't look at the moment.)

**Comment by ValentineHerr** [comment]
I have updated the errors. Just waiting on resolving #91 before pushing

**Comment by teixeirak** [comment]
Thanks, all!

**Comment by jess-shue** [comment]
@teixeirak @emacmonigle @jenajordan 
Hi All,
    Just checking in on this - I'm going to change the background data in the original form 'SCBI Mortality 2022' to match what was in the new form I created last week. That way we'll have all the fields required by Valentine's code and we can filter the FFF's download to that form and download one file.

The data set has 7,480 stems in it. The stems < 10 cm are no longer listed and only stems with a status in the 2021 census of 'A', 'AU', and 'DS' are included. I hope this will remove the stems causing long search times per your criteria at the start of the issue.

**Comment by teixeirak** [comment]
Many thanks, Jess!

**Comment by emacmonigle** [comment]
Great, thanks!

**Comment by emacmonigle** [comment]
Hi all, update to this issue. In the field I've found that PD trees are not always fallen (DC), and are DS around half the time. 

I do have good news that because of these updates (removing missing and PD) have helped me get through many more quads than my previous pace (about 2.5x faster)! Standing PD trees I usually find easily and adding these in I don't think would slow me down too much. Open to everyone's opinions on whether they should be added back or not.

**Comment by teixeirak** [comment]
Great to hear that this speeds things up so much!

It sounds like we should census the PD stems, provided that that doesn't slow things down too much. Of course, any PD that was ever found DC should be removed. @jess-shue  and @ValentineHerr , could you please make this adjustment?

**Comment by jess-shue** [comment]
@teixeirak @emacmonigle 
I'm working with Ryan to collect cores and learn the overall process for the cores as I will be coming to SCBI next week. I'm not sure how quickly I can make this adjustment as it will require some diggin.

**Comment by teixeirak** [comment]
Okay. When we have a chance to get this fixed, some new ones will show up as errors, and we can go back and get them.

**Comment by teixeirak** [comment]
I think we can close this.

**Comment by djj4tree** [comment]
We did not make it out today with the storms.
Yesterday, we noticed that there were standing dead trees that should have
been in the form that were not.  I wanted to make sure that the standing
dead continue to get revisited.

Best, Dan

**Comment by teixeirak** [comment]
@djj4tree , GitHub must have had some glitch-- I received 4 or 5 obviously old messages, including yours above, between 3:30 and 5 AM today. 

I don't know when you actually sent this, but looking back, we removed and then restored trees with a PD (previously dead) status, which could include DS. Hopefully that fixed the issue. We did include all DS trees.
